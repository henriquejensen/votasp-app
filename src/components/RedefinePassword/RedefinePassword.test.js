import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import RedefinePassword from './RedefinePassword';

describe('RedefinePassword', () => {
  it('should match a snapshot', () => {
    expect(toJson(shallow(( <RedefinePassword /> )))).toMatchSnapshot();
  })
})