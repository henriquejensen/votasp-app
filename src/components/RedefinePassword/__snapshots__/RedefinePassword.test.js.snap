// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RedefinePassword should match a snapshot 1`] = `
<FormLayout
  errorMessage={null}
  hideTabs={true}
  successMessage={null}
>
  <h1
    className="redefine-password__title"
  >
    Recuperação de senha
  </h1>
  <form
    onSubmit={[Function]}
  >
    <section
      className="authentication__form-content"
    >
      <div
        className="field-wrapper"
      >
        <label
          htmlFor="email"
        >
          E-mail
        </label>
        <input
          className="input"
          id="email"
          name="email"
          placeholder="Informe o e-mail cadastrado"
          required={true}
          type="text"
        />
      </div>
    </section>
    <button
      className="authentication__submit-button"
      disabled={false}
    >
      Recuperar senha
    </button>
  </form>
</FormLayout>
`;
